lldb/tools/lldb-dap/package.json

```diff
@@ -763,6 +764,16 @@
           }
         ]
       }
-    ]
+    ],
+    "views": {
+      "debug": [
+        {
+          "id": "lldb-dap.modulesExplorer",
+          "name": "LLDB Modules Explorer",
```

Comment:
Is "explorer" common terminology in VS Code for something like this? I've seen the term in the context of a file explorer and a remote (file) explorer but I don't remember seeing it in this context. If not, would "LLDB Modules" or even just "Modules" be descriptive enough for this view?

Reply:
Updated to just be 'Modules'.